client/my-sites/themes/theme-showcase.jsx

```diff
@@ -562,7 +562,6 @@ class ThemeShowcase extends Component {
 		const tiers = this.getTiers();
 
 		const classnames = classNames( 'theme-showcase', {
-			'theme-discovery': this.isThemeDiscoveryEnabled(),
 			'is-discovery-view':
```

Comment:
I think we can drop this as well? 🤔

Reply:
We're not using it anywhere else I guess it's fine. Done.